bokehjs/test/baselines/linux/Legend_annotation__in_vertical_orientation__should_support_grid_layout__with_nrows=auto_and_ncols=3.blf

```diff
@@ -21,5 +21,5 @@ Figure bbox=[0, 0, 300, 300]
     Line bbox=[38, 17, 245, 249]
   LinearAxis bbox=[26, 278, 269, 22]
   LinearAxis bbox=[0, 5, 26, 273]
-  Legend bbox=[49, 15, 236, 83]
-  Legend bbox=[49, 185, 236, 83]
+  Legend bbox=[56, 45, 239, 85]
+  Legend bbox=[56, 213, 239, 85]
```

Comment:
Commenting here since I can't comment on the image. The `fill=even` and `fill=odd` legends on these tests look identical. Are they supposed to differ at all in some way?

Reply:
I forgot to compute canvas relative bbox out of CSS positioning, which is viewport relative.

Reply:
After fixing this, any additional differences a result of:

1. Bokeh's layout not taking borders into consideration.
2. Slight differences in text rendering and alignment between CSS and our text rendering engine.